Formula/fakeroot.rb

```diff
@@ -1,75 +1,41 @@
 class Fakeroot < Formula
   desc "Provide a fake root environment"
   homepage "https://tracker.debian.org/pkg/fakeroot"
-  url "https://deb.debian.org/debian/pool/main/f/fakeroot/fakeroot_1.24.orig.tar.gz"
-  sha256 "2e045b3160370b8ab4d44d1f8d267e5d1d555f1bb522d650e7167b09477266ed"
-  license "GPL-3.0"
+  url "https://deb.debian.org/debian/pool/main/f/fakeroot/fakeroot_1.29.orig.tar.gz"
+  sha256 "8fbbafb780c9173e3ace4a04afbc1d900f337f3216883939f5c7db3431be7c20"
+  license "GPL-3.0-or-later"
 
   bottle do
     sha256 cellar: :any, catalina:    "c72ae187158b6cce73311fee527ba8bf8d2f0e18340bd66eef57b50b3d45c275"
     sha256 cellar: :any, mojave:      "6c23e4c601af569c2de802cac685de5d18e6ebafcb53e6c53107aa3feb3d1527"
     sha256 cellar: :any, high_sierra: "df9be392f3579464893be013744b5aa40a7e4e91e01155bd1547e4104d381640"
   end
 
-  # Does not build. Mac support looks abandoned since 2013
-  # Initial mac support: https://github.com/mackyle/fakeroot
-  # https://salsa.debian.org/clint/fakeroot/-/blob/master/README_MACOSX.txt
-  disable! date: "2022-03-28", because: :does_not_build
+  # Needed to apply patches below. Remove when no longer needed.
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
 
-  on_linux do
-    depends_on "libcap"
-  end
-
-  # Compile is broken. https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=766649
-  # Patches submitted upstream on 24/10/2014, but no reply from maintainer thus far.
+  # https://salsa.debian.org/clint/fakeroot/-/merge_requests/16
   patch do
-    url "https://bugs.debian.org/cgi-bin/bugreport.cgi?msg=5;filename=0001-Implement-openat-2-wrapper-which-handles-optional-ar.patch;att=1;bug=766649"
-    sha256 "1c9a24aae6dc2a82fa7414454c12d3774991f6264dd798d7916972335602308d"
+    url "https://salsa.debian.org/clint/fakeroot/-/commit/e1a7af793e58bddd4bbd04cfb4d26687fbaa9bcf.diff"
+    sha256 "60cfd8bbc416527981151237b7c403fba88975e97907a0ed5c31566d0cda078d"
   end
 
-  patch do
-    url "https://bugs.debian.org/cgi-bin/bugreport.cgi?msg=5;filename=0002-OS-X-10.10-introduced-id_t-int-in-gs-etpriority.patch;att=2;bug=766649"
-    sha256 "e0823a8cfe9f4549eb4f0385a9cd611247c3a11c0452b5f80ea6122af4854b7c"
+  # https://salsa.debian.org/clint/fakeroot/-/merge_requests/17
+  patch :p0 do
+    # The MR has a typo, so we use MacPorts' version.
+    url "https://raw.githubusercontent.com/macports/macports-ports/0ffd857cab7b021f9dbf2cbc876d8025b6aefeff/sysutils/fakeroot/files/patch-message.h.diff"
+    sha256 "6540eef1c31ffb4ed636c1f4750ee668d2effdfe308d975d835aa518731c72dc"
   end
```

Comment:
These are the only patches that MacPorts uses, so it might suffice? https://github.com/macports/macports-ports/tree/0ffd857cab7b021f9dbf2cbc876d8025b6aefeff/sysutils/fakeroot/files